test/sanity/validate-modules/validate-modules

```diff
@@ -569,13 +570,20 @@ class ModuleValidator(Validator):
                             303,
                             'DOCUMENTATION fragment missing: %s' % fragment
                         ))
-                    except Exception as e:
-                        self.traces.append(e)
+                    except Exception:
+                        self.traces.append(traceback.format_exc())
                         self.errors.append((
                             304,
                             'Unknown DOCUMENTATION error, see TRACE'
                         ))
 
+                if 'options' in doc and doc['options'] is None and doc.get('extends_documentation_fragment'):
+                    self.errors.append((
+                        305,
```

Comment:
Technically there is not error message for a 305.  305 is a schema error. I suppose I could, but it feels like the same thing to me.  I just couldn't find an easy way to get voluptuous to do both for me, without more involved effort, otherwise it would be handled there.